packages/alfa-rules/src/sia-r83/rule.ts

```diff
@@ -104,6 +111,36 @@ export default Rule.Atomic.of<Page, Text>({
   },
 });
 
+function hasDisplaySize(
+  valueOrPredicate: number | Predicate<number>
+): Predicate<Element> {
+  const predicate =
+    typeof valueOrPredicate === "function"
+      ? valueOrPredicate
+      : (size: number) => valueOrPredicate === size;
+
+  return (element) => {
+    let displaySize = 0;
+    for (const size of element.attribute("size")) {
+      const valueMenuSize = parseInt(size.value, 10);
+      if (
+        valueMenuSize === valueMenuSize &&
+        valueMenuSize === (valueMenuSize | 0)
+      ) {
+        displaySize = valueMenuSize;
+      }
+    }
+
+    if (element.attribute("multiple").isSome()) {
```

Comment:
`<select size="1" multiple">` get its `size` overwritten. So we need to only do that if no size was computed yet.
Maybe we can chain the option mapper 🤔 
```typescript
const displaySize = element.attribute("size")
  .map(/* parse and check the value*/).
  .getOrElse(() =>element
    .attribute("multiple")
    .map(() => 4)
    .getOr(1)
  )
```